dask/dataframe/io.py

```diff
@@ -798,9 +798,12 @@ def from_delayed(dfs, meta=None, divisions=None, prefix='from-delayed',
         An iterable of ``dask.delayed.Delayed`` objects, such as come from
         ``dask.delayed`` These comprise the individual partitions of the
         resulting dataframe.
-    $META
```

Comment:
Instead of copy-pasting around the parameter description of the meta keyword, I made a decorator that inserts it in functions. This was added in #1422.